jupyterhub/metricshandler.py

```diff
@@ -0,0 +1,18 @@
+from prometheus_client import CONTENT_TYPE_LATEST
+from prometheus_client import generate_latest
```

Comment:
Can we get rid of this extra file and use the existing `jupyterhub/handlers/metrics`?